interop-scoring/main.js

```diff
@@ -0,0 +1,506 @@
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?h264_avc',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?vp8',
+  '/webcodecs/videoDecoder-codec-specific.https.any.html?vp9',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?av1',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?h264_annexb',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?h264_avc',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?vp8',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?vp9',
+  '/webcodecs/videoDecoder-codec-specific.https.any.worker.html?av1',
+  '/webcodecs/videoFrame-construction.any.html',
+  '/webcodecs/videoFrame-construction.crossOriginSource.sub.html',
+  '/webcodecs/videoFrame-construction.window.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/temporal-svc-encoding.https.any.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.html?vp9',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?h264',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp8',
+  '/webcodecs/temporal-svc-encoding.https.any.worker.html?vp9',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/videoFrame-serialization.crossAgentCluster.https.html',
+  '/webcodecs/full-cycle-test.https.any.html?av1',
+  '/webcodecs/full-cycle-test.https.any.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.worker.html?av1',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_annexb',
+  '/webcodecs/full-cycle-test.https.any.worker.html?h264_avc',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p0',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp9_p2',
+  '/webcodecs/full-cycle-test.https.any.html?vp8',
+  '/webcodecs/full-cycle-test.https.any.worker.html?vp8',
+  // interop-2023-webcomponents
+  '/shadow-dom/focus/focus-shadowhost-display-none.html',
+  '/custom-elements/form-associated/ElementInternals-labels.html',
+  '/custom-elements/form-associated/ElementInternals-setFormValue.html',
+  '/custom-elements/form-associated/ElementInternals-validation.html',
+  '/custom-elements/form-associated/form-disabled-callback.html',
+]);
+
+
+// Calculate interop score (passing in all browsers) for a category
+// after tracking the category's scores for each browser.
+function aggregateInteropTestScores(interopScores, numBrowsers) {
+  if (interopScores.length === 0) return 0;
+  let aggregateScore = 0;
+  for (const [, results] of interopScores) {
```

Comment:
I haven't tested this, but something like this should allow iterating just the values:

```suggestion
  for (const results of interopScores.values()) {
```